src/main/java/com/cinema/model/dto/OrderRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package com.cinema.model.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderRequestDto {
+    private Long userId;
```

Comment:
You don't need this, because you request userId from the URL

Reply:
done